### PR TITLE
Update 2021-01-05-Jamulus-Sound-Devices.md

### PR DESCRIPTION
**Pull request by Robert53844**
Added ZOOM H2 as compatible device  and updated hints for drivers for ZOOM H2N & LiveTrak L-8

### CODE REVIEW

File: _posts/2021-01-05-Jamulus-Sound-Devices.md [hunk 1]
```diff
@@ -218,12 +218,14 @@ Sounds great, can achieve 32 frame buffer and works on *Windows* and *Linux*.
...
 
-**Windows**: Latency is around 19ms (measured with local server with ping time of 0 ms) using "L-8 Driver" (ASIO, 07/02/2020) from [zoomcorp.com](https://zoomcorp.com/)
+**Windows**: Latency is around 19ms (measured with local server with ping time of 0 ms) using "L-8 Driver" (ASIO, 07/02/2020) from [zoomcorp.com](https://zoomcorp.com/). Driver can be downloaded from link to device and clicking "Support & Downloads".
 
 **macOS**: Not yet tested
 
 **Linux**: Not yet tested
 
+***
+
```
Comment (ann0see) [inline _posts/2021-01-05-Jamulus-Sound-Devices.md:228]: Thank you. This looks good. I‘m a bit unsure: why did you add a line (`***`) here

**Review by ann0see — COMMENTED** [review]
Thanks

File: _posts/2021-01-05-Jamulus-Sound-Devices.md (hunk 1 above)
Comment (Robert53844) [reply _posts/2021-01-05-Jamulus-Sound-Devices.md:228]: Just to normalize the formatting with a horizontal line between each device. Ignore it, if you think it was not a good idea.

Comment (ann0see) [reply _posts/2021-01-05-Jamulus-Sound-Devices.md:228]: I'm open for suggestions. You could also add it on other places if you wish

**Review by ann0see — APPROVED** [review]
